src/components/pages/Navbar/Navbar.js

```diff
@@ -209,7 +232,6 @@ const Navbar = ({ isAuthenticated, userProfile, getProfile }) => {
 
 const mapStateToProps = state => {
   return {
-    isAuthenticated: localStorage.getItem('token'),
```

Comment:
Excellent job migrating the collection of the token to the useAxiosWithAuth0 hook.